src/nccl_ofi_net.c

```diff
@@ -1361,7 +1366,7 @@ static ncclResult_t ofi_connect(int dev, void *handle, void **sendComm)
 
 	/* Insert remote address into AV */
 	ret = fi_av_insert(nccl_ofi_component[dev]->av,
-			   (void *)remote_ep_addr, 1,
+			   remote_ep_name, 1,
```

Comment:
Intentional removal of type-casting to 'void *' pointer?

Reply:
My apologies, it was my mental auto-pilot removing extra syntax where possible. Restored for your convenience.